packages/react-native-skia/components/textinput/TextInput_Ref.js

```diff
@@ -0,0 +1,63 @@
+import React, {useRef} from "react";
+import { View, StyleSheet,TextInput,AppRegistry,Pressable,TouchableOpacity} from "react-native";
+
+const onFocus = (ne) =>{
+  console.log("onFocus"<< ne);
+ }
+ const onBlur = () =>{
+  console.log("onBlur");
+ }
+const SimpleViewApp = () => {
+  const ref_input1 = useRef();
+  const ref_input2 = useRef();
+  const ref_input3 = useRef();
+  const [text,onChangeText]=React.useState("Useless Text");
+  //setTimeout(()=>{ref_input3.current.blur()},3000);
+  return (
+    <View 
+    onLayout={()=>{ref_input3.current.focus(); console.log("calling textinput 3")}}>
+      <TextInput
+        placeholder="Input1"
+        returnKeyType="next"
+        value="123456"
+        onFocus={()=>{console.log("in First on Focus")}}
+        //onFocus={onFocus}
+        onBlur={()=> {console.log("in First on Blur")}}
+        onEndEditing={console.log("in First on EndEditing")}
+        onSubmitEditing={() => {ref_input2.current.focus(); console.log("calling textinput 2")} }
+        ref={ref_input1}
+      />
+      <TextInput
+        placeholder="Input2"
+        returnKeyType="next"
+        onFocus={()=>{console.log("in second on Focus")}}
+        //onFocus={()=>{onFocus}}
+        onBlur={() => {console.log("in second on Blur")}}
+        onEndEditing={() =>{console.log("in second on EndEditng")}}
+        onSubmitEditing={() => {ref_input3.current.focus(); console.log("calling textinput 3")} }
+        ref={ref_input2}
+      />
+      <TextInput
+        placeholder="Input3"
+        onFocus={()=>{console.log("in Third on Focus")}}
+        //onFocus={() =>{onFocus}}
+        onEndEditing={() =>{console.log("in third on EndEditng")}}
+        onBlur={() =>{console.log("in thir on Blur")}}
+        //onSubmitEditing={()=>{ ref_input1.current.clear();ref_input1.current.focus(); console.log("calling textinput 1")}}
+        ref={ref_input3}
+      />
+    </View>
+  );
+};
+
+const styles = StyleSheet.create({
+  input: {
+    height: 40,
+    margin: 12,
+    borderWidth: 1,
+    padding: 10,
+  },
+});
+
+export default SimpleViewApp;
```

Comment:
Optional:
Is it possible to update the functionalities in this test app to the already existing test apps of text input component.. So that we can improve on additing functionalities in single app and it will be easy for regression tests as well.